app/(public)/[...slug]/page.tsx

```diff
@@ -3,37 +3,33 @@ import {getPathsFromContext} from "@/lib/drupal/get-paths";
 import NodePageDisplay from "@/components/node";
 import {notFound, redirect} from "next/navigation";
 import {translatePathFromContext} from "@/lib/drupal/translate-path";
-import {DrupalMenuLinkContent, DrupalNode} from "next-drupal";
+import {DrupalMenuLinkContent} from "next-drupal";
 import {GetStaticPathsResult, Metadata} from "next";
 import {getNodeMetadata} from "./metadata";
-import Conditional from "@/components/utils/conditional";
 import LibraryHeader from "@/components/node/sul-library/library-header";
-import {Library} from "@/lib/drupal/drupal";
+import {PageProps, Params, StanfordNode} from "@/lib/drupal/drupal";
 import InternalHeaderBanner from "@/components/patterns/internal-header-banner";
-import {Suspense} from "react";
 import SecondaryMenu from "@/components/menu/secondary-menu";
 import {getMenu} from "@/lib/drupal/get-menu";
 import {DrupalJsonApiParams} from "drupal-jsonapi-params";
 import {isDraftMode} from "@/lib/drupal/is-draft-mode";
 import UnpublishedBanner from "@/components/patterns/unpublished-banner";
 
-// Opt out of caching for all data requests in the route segment
-export const dynamic = 'force-dynamic';
-export const revalidate = 0;
+export const revalidate = 2592000;
```

Comment:
Can you tell me more about why you chose 30 days? Why not cache forever?

Reply:
:shrug: Just a number i chose.

Reply:
I figured we had it at 1 day before, this is more cache. If we have no issues with this, we can increase it to infinite